exporter/opentelemetry-exporter-otlp-proto-grpc/tests/metrics/test_otlp_metrics_exporter.py

```diff
@@ -112,23 +117,140 @@ def setUp(self):
+                                ),
+                                metrics=[_generate_sum("sum_double", 2.98)],
+                                schema_url="instrumentation_scope_schema_url",
+                            )
+                        ],
+                        schema_url="resource_schema_url",
+                    )
+                ]
+            ),
+            "gauge_int": MetricsData(
+                resource_metrics=[
+                    ResourceMetrics(
+                        resource=Resource(
+                            attributes={"a": 1, "b": False},
+                            schema_url="resource_schema_url",
+                        ),
+                        scope_metrics=[
+                            ScopeMetrics(
+                                scope=SDKInstrumentationScope(
+                                    name="first_name",
+                                    version="first_version",
+                                    schema_url="insrumentation_scope_schema_url",
+                                ),
+                                metrics=[_generate_gauge("gauge_int", 9000)],
+                                schema_url="instrumentation_scope_schema_url",
+                            )
+                        ],
+                        schema_url="resource_schema_url",
+                    )
+                ]
+            ),
+            "gauge_double": MetricsData(
+                resource_metrics=[
+                    ResourceMetrics(
+                        resource=Resource(
+                            attributes={"a": 1, "b": False},
+                            schema_url="resource_schema_url",
+                        ),
+                        scope_metrics=[
+                            ScopeMetrics(
+                                scope=SDKInstrumentationScope(
+                                    name="first_name",
+                                    version="first_version",
+                                    schema_url="insrumentation_scope_schema_url",
+                                ),
+                                metrics=[
+                                    _generate_gauge("gauge_double", 52.028)
+                                ],
+                                schema_url="instrumentation_scope_schema_url",
+                            )
+                        ],
+                        schema_url="resource_schema_url",
+                    )
+                ]
+            ),
+            "histogram": MetricsData(
+                resource_metrics=[
+                    ResourceMetrics(
+                        resource=Resource(
+                            attributes={"a": 1, "b": False},
+                            schema_url="resource_schema_url",
+                        ),
+                        scope_metrics=[
+                            ScopeMetrics(
+                                scope=SDKInstrumentationScope(
```

Comment:
I would like to see some test contain multiple scopes here. Like `scope_1`, `scope_2` and `scope_1` again, `scope_3`.